app/ui/browser/actions/external.js

```diff
@@ -227,7 +228,7 @@ export function close() {
  * Perform the search if the user just pressed return
  */
 export function inPageSearch(ev) {
-  if (ev.keyCode === 13) {
+  if (ev.keyCode === KEY.ENTER) {
```

Comment:
I think it's a little easier since we won't need to add a new constant every time we handle a new key.  Not sure about cross platform issues but seems like there shouldn't be any